ldapjs/README.md

```diff
@@ -0,0 +1,6 @@
+# LDAP Server
+
+This basic LDAP server is to be used for testing purposes only.
+It is not meant for production use.
+
+Please see the OpenLDAP implementations for production-ready servers.
```

Comment:
Why shouldn't we use OpenLDAP for testing? They do have a docker image.

Reply:
To be honest, I think using OpenLDAP may be overkill, so I spun up a lightweight NodeJS implementation of an LDAP server. Both are essentially equivalent, and the HA is going to fine-tune the passport settings anyway.